### PR TITLE
Index html

### PR DESCRIPTION
**Pull request by HTMLissa**

### CODE REVIEW

**Review by JoyBoy92Codes — COMMENTED** [review]
Changes Approved, looks good!

File: index.html [hunk 1]
```diff
@@ -69,48 +69,54 @@ <h1>Dental Clinic</h1>
...
-        </ul>
-        </p>
-      </div>
-      <div class="service-box service-box-white">
-        <img src="images/toothbrushes.jpg" alt="Wooden toothbrushes in a jar" class="service-box-img">
-      </div>
-      <div class="service-box service-box-white">
-        <img src="images/woman-holding-dentist-tools.jpg" alt="Woman holding dentist tools while smiling"
-          class="service-box-img">
+    <article class="services">
```
Comment (JoyBoy92Codes) [inline index.html]: Update article tag to section tag

**Review by JoyBoy92Codes — COMMENTED** [review]
Update tag

File: index.html (hunk 1 above)
Comment (HTMLissa) [reply index.html]: Done

**Review by JoyBoy92Codes — COMMENTED** [review]
Looks good to me, great work

**Pull request merged**